tests/test_constants_and_functions.cpp

```diff
@@ -75,8 +77,10 @@ struct C {
 #  pragma GCC diagnostic push
 #  pragma GCC diagnostic ignored "-Wdeprecated"
 #endif
-    int m7(int x) throw() { return x-7; }
-    int m8(int x) const throw() { return x-8; }
+    // NOLINTNEXTLINE(modernize-use-noexcept)
```

Comment:
This check discourages the user of throw, but since these are explicilty deprecated tests, I add nolints to prevent them from being changed.